website/src/pages/contact.tsx

```diff
@@ -1,6 +1,7 @@
 import Layout from "@theme/Layout";
 import { Calendar, Github, Mail, Slack } from "lucide-react";
 import React from "react";
+import validator from "validator";
```

Comment:
Instead of bringing in a new dep could we maybe just check whether the trimmed field value is empty (and for email, just make sure it contains "@" ?)